src/sentry/demo/demo_start.py

```diff
@@ -31,7 +31,10 @@ def post(self, request):
         logger.info("post.start", extra={"cookie_member_id": member_id})
         sentry_sdk.set_tag("member_id", member_id)
 
-        if member_id:
+        skip_buffer = request.POST.get("skip_buffer") == "1"
+        sentry_sdk.set_tag("skip_buffer", skip_buffer)
+
+        if member_id and not skip_buffer:
```

Comment:
When we skip the buffer, we also don't want to reassign the user to the org based on the cookie.

Reply:
Do we care if end users poke the demo server with skip_buffer=1 often?

Reply:
@markstory It wouldn't cause any problems, just a less ideal user experience.